### ISSUE TITLE
Blank NDI Studio Monitor Output

### ISSUE THREAD

**Issue by rumplepusscat** [Rushmead/MC-NDI#1]
Hi there,

I have a small amount of experience with NDI, so was incredibly excited when hearing about this!

I'm using NDI Studio Monitor, and while the individual cameras/main shots are showing up in the source list, selecting them only shows a black screen.

I'm running MC 1.16.5, Forge 36.0.55 & mod version 1.0.0

Many thanks!

Rumple :)

**Comment by rumplepusscat** [comment]
Apologies for not posting before - here is my log, as requested:

```
[15Mar2021 17:40:46.512] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, Rumplepusscat, --version, 1.16.5-forge-36.0.55, --gameDir, C:\Users\bened\AppData\Roaming\.minecraft, --assetsDir, C:\Users\bened\AppData\Roaming\.minecraft\assets, --assetIndex, 1.16, --uuid, dd360176e94b4c5fa8e4630851ba791a, --accessToken, ????????, --userType, mojang, --versionType, release, --launchTarget, fmlclient, --fml.forgeVersion, 36.0.55, --fml.mcVersion, 1.16.5, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20210115.111550]
[15Mar2021 17:40:46.515] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 8.0.9+86+master.3cf110c starting: java version 1.8.0_51 by Oracle Corporation
[15Mar2021 17:40:46.527] [main/WARN] [cpw.mods.modlauncher.SecureJarHandler/]: LEGACY JDK DETECTED, SECURED JAR HANDLING DISABLED
[15Mar2021 17:40:46.829] [main/INFO] [net.minecraftforge.fml.loading.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[15Mar2021 17:40:46.873] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/bened/AppData/Roaming/.minecraft/libraries/org/spongepowered/mixin/0.8.2/mixin-0.8.2.jar Service=ModLauncher Env=CLIENT
[15Mar2021 17:40:48.542] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'fmlclient' with arguments [--version, 1.16.5-forge-36.0.55, --gameDir, C:\Users\bened\AppData\Roaming\.minecraft, --assetsDir, C:\Users\bened\AppData\Roaming\.minecraft\assets, --uuid, dd360176e94b4c5fa8e4630851ba791a, --username, Rumplepusscat, --assetIndex, 1.16, --accessToken, ????????, --userType, mojang, --versionType, release]
[15Mar2021 17:40:55.423] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[15Mar2021 17:40:56.138] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Setting user: Rumplepusscat
[15Mar2021 17:40:56.242] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Backend library: LWJGL version 3.2.2 build 10
[15Mar2021 17:40:57.054] [modloading-worker-3/INFO] [STDOUT/]: [dev.imabad.ndi.NDIMod:init:31]: Starting Fabric NDI, loading NDI libraries.
[15Mar2021 17:40:57.057] [modloading-worker-2/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 36.0.55, for MC 1.16.5 with MCP 20210115.111550
[15Mar2021 17:40:57.081] [modloading-worker-2/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v36.0.55 Initialized
[15Mar2021 17:40:59.139] [Render thread/INFO] [com.mojang.text2speech.NarratorWindows/]: Narrator library for x64 successfully loaded
[15Mar2021 17:40:59.272] [Render thread/INFO] [net.minecraft.resources.SimpleReloadableResourceManager/]: Reloading ResourceManager: Mod Resources, Default
[15Mar2021 17:40:59.801] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/maven/net/minecraftforge/forge/promotions_slim.json
[15Mar2021 17:41:01.399] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: BETA_OUTDATED Current: 36.0.55 Target: 36.0.58
[15Mar2021 17:41:04.962] [Render thread/INFO] [net.minecraft.client.audio.SoundSystem/]: OpenAL initialized.
[15Mar2021 17:41:04.963] [Render thread/INFO] [net.minecraft.client.audio.SoundEngine/SOUNDS]: Sound engine started
[15Mar2021 17:41:05.250] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[15Mar2021 17:41:05.343] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[15Mar2021 17:41:05.343] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[15Mar2021 17:41:05.353] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[15Mar2021 17:41:05.354] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[15Mar2021 17:41:05.354] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[15Mar2021 17:41:05.355] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[15Mar2021 17:41:06.029] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[15Mar2021 17:41:06.051] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[15Mar2021 17:41:06.051] [Render thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[15Mar2021 17:41:28.304] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[15Mar2021 17:41:28.305] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:28.305] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:28.308] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[15Mar2021 17:41:28.309] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:28.312] [Render thread/INFO] [net.minecraft.resources.SimpleReloadableResourceManager/]: Reloading ResourceManager: Default, MC-NDI-1.0.0-forge.jar, forge-1.16.5-36.0.55-universal.jar, architectury-1.8.131-forge.jar
[15Mar2021 17:41:28.841] [Render thread/INFO] [net.minecraft.item.crafting.RecipeManager/]: Loaded 7 recipes
[15Mar2021 17:41:29.315] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 927 advancements
[15Mar2021 17:41:29.425] [Render thread/WARN] [net.minecraftforge.fml.FMLWorldPersistenceHook/WP]: This world was saved with mod forge version 36.0.58 and it is now at version 36.0.55, things may not work well
[15Mar2021 17:41:30.031] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[15Mar2021 17:41:30.031] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:30.032] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:30.033] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[15Mar2021 17:41:30.033] [Render thread/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[15Mar2021 17:41:30.034] [Render thread/INFO] [net.minecraft.resources.SimpleReloadableResourceManager/]: Reloading ResourceManager: Default, MC-NDI-1.0.0-forge.jar, forge-1.16.5-36.0.55-universal.jar, architectury-1.8.131-forge.jar
[15Mar2021 17:41:30.425] [Render thread/INFO] [net.minecraft.item.crafting.RecipeManager/]: Loaded 7 recipes
[15Mar2021 17:41:30.679] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 927 advancements
[15Mar2021 17:41:30.698] [Render thread/WARN] [net.minecraftforge.fml.FMLWorldPersistenceHook/WP]: This world was saved with mod forge version 36.0.58 and it is now at version 36.0.55, things may not work well
[15Mar2021 17:41:31.437] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[15Mar2021 17:41:31.468] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Starting integrated minecraft server version 1.16.5
[15Mar2021 17:41:31.468] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Generating keypair
[15Mar2021 17:41:31.656] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Preparing start region for dimension minecraft:overworld
[15Mar2021 17:41:32.487] [Render thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[15Mar2021 17:41:32.488] [Render thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[15Mar2021 17:41:32.788] [Render thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 55%
[15Mar2021 17:41:33.024] [Render thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Time elapsed: 1315 ms
[15Mar2021 17:41:33.256] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Changing view distance to 15, from 10
[15Mar2021 17:41:34.544] [Netty Local Client IO #0/INFO] [net.minecraftforge.fml.network.NetworkHooks/]: Connected to a modded server.
[15Mar2021 17:41:34.588] [Server thread/INFO] [net.minecraftforge.common.AdvancementLoadFix/]: Using new advancement loading for net.minecraft.advancements.PlayerAdvancements@174716de
[15Mar2021 17:41:34.600] [Server thread/INFO] [net.minecraft.server.management.PlayerList/]: Rumplepusscat[local:E:4c75e9f1] logged in with entity id 254 at (215.04401917786484, 73.0, 6.67152173821639)
[15Mar2021 17:41:34.622] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Rumplepusscat joined the game
[15Mar2021 17:41:34.958] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Saving and pausing game...
[15Mar2021 17:41:34.990] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:overworld
[15Mar2021 17:41:35.349] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 0 advancements
[15Mar2021 17:41:35.843] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:the_nether
[15Mar2021 17:41:35.844] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:the_end
[15Mar2021 17:46:33.747] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Saving and pausing game...
[15Mar2021 17:46:33.760] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:overworld
[15Mar2021 17:46:33.765] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:the_nether
[15Mar2021 17:46:33.765] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[NDI]'/minecraft:the_end
```

**Comment by Rushmead** [comment]
Fixed in 1.0.1! Thanks for reporting :) 

https://www.curseforge.com/minecraft/mc-mods/mc-ndi/files/3240547
